packages/config/config/devices/0x019a/11_04_022.json

```
@@ -0,0 +1,233 @@
+{
+	"manufacturer": "Sensative AB",
+	"manufacturerId": "0x019a",
+	"label": "11-01-021",
+	"description": "Strips Drip 700",
+	"devices": [
+		{
+			"productType": "0x0004",
+			"productId": "0x000b"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"2": {
+			"label": "LED Alarm Event Reporting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "LED Off",
+					"value": 0
+				},
+				{
+					"label": "LED On",
+					"value": 1
+				}
+			]
+		},
+		"4": {
+			"label": "Temperature Reporting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				}
```

```suggestion
				{
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Enable",
					"value": 1
				}
```
please carry througout